subsys/net/ip/ipv4_autoconf.c

```diff
@@ -168,12 +168,12 @@ static void ipv4_autoconf_send(struct net_if_ipv4_autoconf *ipv4auto)
 		ipv4auto->announce_cnt = 0U;
 		ipv4auto->conflict_cnt = 0U;
 		(void)memset(&ipv4auto->current_ip, 0, sizeof(struct in_addr));
-		ipv4auto->requested_ip.s4_addr[0] = 169U;
-		ipv4auto->requested_ip.s4_addr[1] = 254U;
-		ipv4auto->requested_ip.s4_addr[2] = sys_rand32_get() % 254;
+		ipv4auto->requested_ip.s4_addr[0] = 192U;
+		ipv4auto->requested_ip.s4_addr[1] = 168U;
+		ipv4auto->requested_ip.s4_addr[2] = 0U;
 		ipv4auto->requested_ip.s4_addr[3] = sys_rand32_get() % 254;
 
-		NET_DBG("%s: Starting probe for 169.254.%d.%d", "Init",
+		NET_DBG("%s: Starting probe for 192.168.%d.%d", "Init",
```

Comment:
Nack to this change. The IPv4 auto addresses are from 169.254/16 subnet. See https://tools.ietf.org/html/rfc3927 for details.

Reply:
Thanks for the feedback. Indeed I've used this change for testing.

Reply:
I will split this patch this set and fix issues indicated by gitlint, sanitycheck and uncrustify.